src/scrapers/platformen/mijnbuurtje.py

```diff
@@ -0,0 +1,180 @@
+        """
+         Parameters
+        ----------
+        location : str
+            The name of the main region this mijn buurtje instance is about.
+        """
+
+        super().__init__(platform_url, list_endpoint, details_endpoint)
+        self.location = location
+
+
+class MijnBuurtjeSource(PlatformSource):
+    ITEM_SCHEMA = {'name': {"xpath": "//div/h1[1]/text()"},
+                   # 'orig_group': {
+                   #     'xpath': '//span[@class="mb-help-request meta-item-icon"]/following-sibling::span[1]/text()'},
+                   'group': {
+                       'xpath': '//span[@class="mb-help-request meta-item-icon"]/following-sibling::span[1]/text()',
+                       'transform': lambda text: MijnBuurtjeSource.format_group(text)},
+                   'description': {
+                       "xpath": "//div[contains(@class, 'content-section')][3]/*/node()",
+                       'all': True,
+                       'transform': lambda elements: MijnBuurtjeSource.recursive_text(elements)},
+                   'organiser': {'xpath': "//a[@class='entity']/div[contains(@class, 'entity-content')]/div[contains("
+                                          "@class, 'entity-content-title')]/text()",
+                                 'transform': lambda text: MijnBuurtjeSource.format_organizer(text)},
+                   # 'organiser_kind': {'xpath': '//span[@class="meta-item-content" and contains(text(),"Vraag vanuit:")]',
+                   #               'transform': lambda text: text.replace("Vraag vanuit: ", "")},
+                   'category': {
+                       'xpath': '//span[@class="meta-item-content" and contains(text(),"Thema:")]',
+                       'transform': lambda elem: MijnBuurtjeSource.strip_text(elem, "Thema: ")},
+                   'frequency': {
+                       "xpath": "//span[@class='mb-calendar meta-item-icon']/following-sibling::span[1]/text()",
+                       "transform": lambda elem: MijnBuurtjeSource.json_frequency(elem)},
+                   'created_at': {
+                       'xpath': '//div[@class="heading3 heading3--semibold"]/text()',
+                       'transform': lambda text: MijnBuurtjeSource.format_date(text)},
+                   'location': {
+                       'xpath': '//span[@class="meta-item-content" and contains(text(),"Dorp:")]',
+                       'transform': lambda elem: MijnBuurtjeSource.strip_text(elem, "Dorp: ")},
+                   }
+
+    def __init__(self, config: MijnBuurtjeSourceConfig):
+        super().__init__(config)
+        self.item_parser = TreeParser(None, None, self.ITEM_SCHEMA)
+
+    def initiatives(self) -> Generator[InitiativeImport, None, None]:
+        page_counter = 1
+        try:
+            while page_counter < 100:
+                list_page_url = self.config.list_endpoint + f"&page={page_counter}"
```

Comment:
Met `requests` kun je iets makkelijker werken met GET params, in een dict enzo. Nu is het een beetje string munging :P
Weet niet of dat hier heel handig is, want dan zou je wel wat helper methods moeten toevoegen in de parent class.

Reply:
Shit, wilde deze nog niet resolven. Maar ik heb bestaande code hier wat uitgebreid en aangepast. Ik zie nog niet direct een noodzaak tot een dependency nu voor die paar functies. Als het meer nodig os zou ik het wel overwegen. Maar ik weet niet of ik überhaupt met die TreeParser zou blijven werken in andere scrapers.

Reply:
Ik denk eerder dat een url class oid op z'n plek is. Alleen als je dit goed wilt doen zou je de url builder methods een replace/fill moeten laten doen van zowel query params als url segments wil je dit helemaal compleet maken. Ik heb wat commentaar in de code gezet, maar het is wel even wat meer dan enkel requests anders gebruiken, mede omdat definitie van die urls niet op dezelfde plek gebeurt.